tests/components/deconz/test_services.py

```diff
@@ -52,7 +67,7 @@ async def test_service_setup(hass):
     ) as async_register:
         await deconz.services.async_setup_services(hass)
         assert hass.data[deconz.services.DECONZ_SERVICES] is True
-        assert async_register.call_count == 2
+        assert async_register.call_count == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6259301)